.github/workflows/e2e-test.yml

```diff
@@ -147,6 +147,49 @@ jobs:
           --spec "$SPEC_PATH" \
           --browser ${{ steps.cypress-prep.outputs.chrome-path }}
 
+      - name: Extract chunk-specific timings
+        id: extract-timings
+        if: inputs.split_index >= 0
+        shell: bash
+        run: |
+          echo "=== EXTRACTING CHUNK-SPECIFIC TIMINGS ==="
+
+          # Extract the JSON timing data that cypress-split outputs for this chunk
+          echo "Looking for cypress-split timing output in logs..."
+
+          if ! grep -q "cypress-split: here are passing spec timings" cypress-output.log; then
+            echo "❌ ERROR: cypress-split timing output not found in logs"
+            exit 1
+          fi
+
+          echo "✅ Found cypress-split timing output"
+
+          # Extract the JSON block that follows "cypress-split: here are passing spec timings"
+          CHUNK_TIMING_JSON=$(sed -n '/cypress-split: here are passing spec timings/,/^}/p' cypress-output.log | tail -n +2)
```

Comment:
This part isn't that nice. It's pretty much parsing the output to get the timings. The problem is I need to know which tests actually ran. split-merge results in a file with timings for all tests, even if they didn't run in this chunk.

Reply:
I didn't explain very well - cypress-split does create `newTimes.json`, but it has timings for all specs, whether or not they ran in this chunk.

The problem is, if every chunk uploads a timing file with all specs, how would the parent workflow know how to merge that together? It needs to know what timings came from which spec. The only place I would find this available was in the log.

There's probably a better way to find which specs ran by setting an env/output from the cypress runners?

Reply:
I changed it so it now compares the old timings file to the one cypress-split generates, and then uploads the ones that have changed in the chunk. Then these all get merged across each chunk